lib/engine/game/g_system18/step/dividend.rb

```diff
@@ -12,7 +12,7 @@ class Dividend < Engine::Step::Dividend
           include Engine::Step::HalfPay
 
           def share_price_change(entity, revenue = 0)
-            return super unless @game.game_capitalization == :incremental
+            return super if @game.share_price_change_as_full_cap_by_map?
```

Comment:
This seems to be conflating price changes on issuing shares with price changes on dividends. I'd prefer to see separate game methods for each "control".

Reply:
Good point, it is cleaner to have these "controls" separate.

I committed an improved version (using a slightly different approach) that separates these controls.

Thank you!